### PR TITLE
Refactor calculator and app classes to function components

### PR DESCRIPTION
**Pull request by danmainah**
### Math Magicians: refactor with hooks

**In this milestone I refactored the  calculator and app classes to function components**

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,69 +1,61 @@
...
-      operation: null,
-    };
-    this.handleCalculation = this.handleCalculation.bind(this);
-  }
+function Calculator() {
+  const [state, setState] = useState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
```
Comment (Emmyn5600) [inline src/components/Calculator.js]: - [x] Let us use ES6 syntax, kindly refactor with an arrow function instead of function

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @danmainah 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have been followed.:heavy_check_mark:
- Project is working properly:heavy_check_mark:
calcc

Changes required :writing_hand: 
  - Remove the commented line in the code that has been pushed.
  - Use Es6 syntax instead of function use arrow function
  calcc2
  
  - Please check the comment under the review.

  Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Calculator.js (hunk 1 above)
Comment (danmainah) [reply src/components/Calculator.js]: Thank you for the feedback,I have implemented changes requested

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @danmainah

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You have a descriptive Readme file:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer. 
 
 #### Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**